### PR TITLE
KE-40948 Add rowGroup filters info

### PR DESCRIPTION
**Pull request by yabola**

### CODE REVIEW

**Comment by yabola** [comment]
image

File: sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala [hunk 1]
```diff
@@ -266,6 +266,7 @@ class ParquetFileFormat
       val sharedConf = broadcastedHadoopConf.value.value
 
       S3FileUtils.tryOpenClose(sharedConf, filePath)
+      val time1 = System.currentTimeMillis()
```
Comment (7mming7) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala:269]: 打印时间 logTime func

**Comment by yabola** [comment]
image

**Comment by yabola** [comment]
image

**Review by 7mming7 — APPROVED** [review]
LGTM

**Pull request merged**